examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -1374,12 +1422,19 @@ def main(args):
             text_encoder_two.gradient_checkpointing_enable()
 
     # now we will add new LoRA weights to the attention layers
+
+    if args.use_blora:
+        # if using B-LoRA adding weight only to two blocks
+        target_modules = get_blora_target_modules(unet)
+    else:
+        target_modules = ["to_k", "to_q", "to_v", "to_out.0"]
+
     unet_lora_config = LoraConfig(
         r=args.rank,
-        lora_alpha=args.rank,
         use_dora=args.use_dora,
+        lora_alpha=args.rank,
         init_lora_weights="gaussian",
```

Comment:
why are we using gaussian unconditionally? i found that `default` performs better and has fewer artifacts earlier on.